modules/modelSetup/BasePixArtAlphaSetup.py

```diff
@@ -40,20 +40,20 @@ class BasePixArtAlphaSetup(
 ):
 
     def __init__(self, train_device: torch.device, temp_device: torch.device, debug_mode: bool):
-        super(BasePixArtAlphaSetup, self).__init__(train_device, temp_device, debug_mode)
+        super().__init__(train_device, temp_device, debug_mode)
 
     def setup_optimizations(
             self,
             model: PixArtAlphaModel,
             config: TrainConfig,
     ):
         if config.attention_mechanism == AttentionMechanism.DEFAULT:
-            for name, child_module in model.transformer.named_modules():
+            for _name, child_module in model.transformer.named_modules():
```

Comment:
The `_name` can maybe be fixed up/removed, a subject for another PR.

Reply:
I've already done that in a few places by replacing `named_modules()` with `modules()`. not sure why I used `named_modules()` in the first place. maybe for debugging, then forgot to remove it again.